src/compiler/utilitiesPublic.ts

```diff
@@ -1074,8 +1074,12 @@ namespace ts {
         return SyntaxKind.FirstLiteralToken <= kind && kind <= SyntaxKind.LastLiteralToken;
     }
 
+    export function isBooleanLiteral(node: Node): node is BooleanLiteral {
+        return node.kind === SyntaxKind.TrueKeyword || node.kind === SyntaxKind.FalseKeyword;

```

Comment:
I wish we could move TrueKeyword inside the First/LastLiteralToken range, but it would probably not be worth the breaks it would cause.